geotrek/trekking/locale/fr/LC_MESSAGES/django.po

```diff
@@ -240,8 +243,18 @@ msgstr "Zone de sensibilité"
 msgid "POIs"
 msgstr "POIs"
 
-msgid "services"
-msgstr "services"
+msgid "Services"
+msgstr "Services"
+
+#, fuzzy
+#| msgid "Stages"
+msgid "Signages"
+msgstr "Étapes"
+
+#, fuzzy
+#| msgid "Disabled Infrastructure"
+msgid "Infrastructures"
+msgstr "Aménagement handicapés"
```

Comment:
Shouldn't it be "Disabled infrastructures" as initially, instead of "Infrastructures" ?